src/main/java/org/broadinstitute/hellbender/tools/walkers/annotator/VariantAnnotator.java

```diff
@@ -164,18 +153,16 @@
     @Argument(fullName="resource-allele-concordance", shortName="rac", doc="Check for allele concordances when using an external resource VCF file", optional=true)
     protected Boolean expressionAlleleConcordance = false;
 
-    /**
-     * You can use the -XA argument in combination with this one to exclude specific annotations.Note that some
-     * annotations may not be actually applied if they are not applicable to the data provided or if they are
-     * unavailable to the tool (e.g. there are several annotations that are currently not hooked up to
-     * HaplotypeCaller). At present no error or warning message will be provided, the annotation will simply be
-     * skipped silently. You can check the output VCF header to see which annotations were actually applied (although
-     * this does not guarantee that the annotation was applied to all records in the VCF, since some annotations have
-     * additional requirements, e.g. minimum number of samples or heterozygous sites only -- see the documentation
-     * for individual annotations' requirements).
-     */
-    @Argument(fullName="use-all-annotations", shortName="all", doc="Use all possible annotations (not for the faint of heart)", optional=true)
-    protected Boolean USE_ALL_ANNOTATIONS = false;
+    public List<ReadFilter> getDefaultReadFilters() {
+        return Lists.newArrayList( new WellformedReadFilter(),
+                ReadFilterLibrary.NOT_DUPLICATE,
+                ReadFilterLibrary.PRIMARY_LINE,
+                ReadFilterLibrary.PASSES_VENDOR_QUALITY_CHECK,
+                ReadFilterLibrary.MAPPED);
+    }
+
+    @Override
+    public boolean useVariantAnnotations() { return true;}
```

Comment:
I assume that in this case, the lack of an override for the `getDefaultVariantAnnotation*` methods is intentional? If so, add a comment explaining that by default this tool does not have any annotations turned on.